daily_data/index.html.erb

```diff
@@ -6,7 +6,7 @@
   <% end %>
 <% end %>
 
-<%= form_tag("/add/data") do %>
+<%= form_tag("/daily_data") do %>
```

Comment:
Once the URL helpers are available in the controller, they will (like the controller’s instance variables) be magically available here too.